plugins/t/check_smtp.t

```diff
@@ -16,6 +16,8 @@ my $host_tcp_smtp_nostarttls = getTestParameter( "NP_HOST_TCP_SMTP_NOSTARTTLS",
 					   "A host providing SMTP without STARTTLS", "");
 my $host_tcp_smtp_tls        = getTestParameter( "NP_HOST_TCP_SMTP_TLS",
 					   "A host providing SMTP with TLS", $host_tcp_smtp);
+my $host_tls_cert            = getTestParameter( "NP_HOST_TLS_CERT",
```

Comment:
I like the additonal sanity check, but not the name of this variable. It does not obviously tell me, that this is the CN in the certificate.
Maybe `certificate_common_name` or something like that?
What do you think?

Reply:
agreed.